jatszohaz/jatszohaz/views.py

```diff
@@ -183,3 +187,26 @@ def get(self, request, token, *args, **kwargs):
         login(request, user)
         messages.info(request, _("Logged in as user %s.") % str(user))
         return redirect("/")
+
+
+class NewCommentView(LoginRequiredMixin, FormView):
+    form_class = NewCommentForm
+
+    def get_object(self):
+        return get_object_or_404(JhUser, pk=self.kwargs['user_pk'])
+
+    def get(self, request, *args, **kwargs):
+        return redirect(self.get_object().get_absolute_url())
+
+    def form_valid(self, form):
+        user = self.get_object()
+        member = self.request.user
+        message = form.cleaned_data['comment']
+
+        comment = Comment.objects.create(user=user, member=member, message=message)

```

Comment:
```suggestion
        Comment.objects.create(user=user, member=member, message=message)
```

Ha úgyse használod fel a változót, fölösleges eltárolni.